int8 eStop
int8 homed
int8 resting
int8 running
int8 gripper_closed

int64[6] encoder_pulses
float64[6] joint_angles
float64[6] setpoint_angles



